cairo/tests/ethereum/cancun/vm/instructions/test_arithmetic.py

```diff
@@ -38,7 +38,7 @@ def test_sub(self, cairo_run, evm: Evm):
         try:
             cairo_result = cairo_run("sub", evm)
         except Exception as cairo_error:
-            with pytest.raises(type(cairo_error)):
+            with assert_raises_exactly(type(cairo_error)):
```

Comment:
why doesn't pytest.raises assert 'strictly' on the `type(cairo_error)`?

Reply:
i dont know what to answer you but approve me

Reply:
maybe there's an easier wrapper? nit at this point but worth a look
<img width="1082" alt="image" src="https://github.com/user-attachments/assets/0f9280ab-8ac5-4f4e-b2df-93be0b68d528" />

Reply:
https://docs.pytest.org/en/stable/reference/reference.html#pytest.raises

Reply:
The context manager produces an [ExceptionInfo](https://docs.pytest.org/en/stable/reference/reference.html#pytest.ExceptionInfo) object which can be used to inspect the details of the captured exception:

Reply:
best I can do is a wrapper around pytest's raise, which clearly says